lexers/http.go

```diff
@@ -0,0 +1,101 @@
+package lexers
+
+import (
+	. "github.com/alecthomas/chroma" // nolint
+	"strings"
+)
+
+var httpBodyContentType string
```

Comment:
Unfortunately this won't be concurrent safe. You'll have to create a custom lexer that wraps the regex lexer and `Lexer.Tokenise(...)`. Your returned custom iterator will have to contain this state.

This is actually relatively straightforward. You can see an example of a stateful iterator in `coalesce.go`.